README.md

```diff
@@ -117,7 +117,7 @@ export default Ember.Controller.extend({
 ## FullCalendar Scheduler
 
 ### Opting In
-By default, FullCalendar Scheduler is NOT imported. To include it, add the following to your application's `ember-cli-build.js`:
+By default, FullCalendar Scheduler is NOT imported. To include it, add the following to your application's `config/environment.js`:
```

Comment:
Interesting. This works for me as expected, but I'd have to check to see if there's any difference in other versions of Ember.